demos/heat/demo_heat_dirk.py.rst

```diff
@@ -0,0 +1,126 @@
+Solving the Heat Equation with Irksome
+======================================
+
```

Comment:
Should we note that this is the DIRK demo in the title block?  Seems clear from the filename, of course, but would be nice to emphasize the distinction a bit more.

Reply:
Ok